apps/dashboard/app/controllers/ws/ws_applications_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Ws::WsApplicationsController < ApiController
```

Comment:
I think I want an `/api/v1` type feel.  Looks like the files routes are `files/api/v1/fs/*filepath` but even those I'd like to migrate to `/api/v1/fs/*filepath` and remove the prefix so we have some sort of consistency everything.

Reply:
I have updated to reflect the comments.
Please, let me know what do you think?

Reply:
That works just fine for me, thanks!